### PR TITLE
[Ready for Review] Collab cafe notes - Jan-Apr 2024

### PR DESCRIPTION
**Pull request by aleesteele**
### Summary

Redo of #3487 - adding collab cafe notes from january, february, march, april 2024

Importantly, I removed the 'check-in' question answers for folks, after speaking with our Data Protection Officer at the Turing!

### CODE REVIEW

**Review by KirstieJane — COMMENTED** [review]
Thanks for archiving @aleesteele !

I've left a few formatting comments and I have an open question about why we're dropping the check-in questions. I'd prefer to keep them as they demonstrate the creativity and diversity in our attendees!

Might be something we have to fix forwards by drawing attendees attention to the point about the notes being archived in github and encouraging them to use the 🤫 emoji if there are answers they don't want to be stored here.

**Comment by aleesteele** [comment]
Thanks so much for the notes & review @KirstieJane!

Haven't forgotten about this! I'm speaking with Kit tomorrow at his drop-in hours to do a final review on the check-in question vs name for DP. We're getting this policy, which I'll make sure to drop in #3534 as well. Then I can get these notes updated and pushed for another review before we merge 😄

File: governance/community-calls/20240103-collaboration-cafe.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# Check-in
+
+Name + Pronouns + What are you going to do in 2024 FOR THE PLOT (aka. to further the story of yout life) + an emoji (if you'd like!: https://emojipedia.org/, see also: https://openmoji.org/)
+(Remember that this is a public document. You can use a pseudonym if you'd prefer.) Use the 🤫 emoji if you would not like to be included in our public archive. These notes will be archived on The Turing Way repository.
+
+* Anne
+* Sophia
+* Danny
+
+## Advertise and promote your event or anything exciting you're working on. ✨
```
[the remark this answers is not in the document]
Comment (aleesteele) [reply governance/community-calls/20240103-collaboration-cafe.md]: Weird - yep, fixing this formatting

File: governance/community-calls/20240221-collaboration-cafe.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+* Thinking about proposing a markdown flavour for protocols - inspired by the format of protocols.io - maybe some kind of extension to MyST (Richard J. Acton)
+*  I am moderating this upcoming panel on "Queer Representations of AI", which I'm excited about: [https://www.sas.ac.uk/events/queer-representations-ai](https://www.sas.ac.uk/events/queer-representations-ai) (Kalle)
+    * So cool!
+
+## Breakout rooms: Topic proposals and notes✨ 
+While no sign-ups are required to attend Collaboration Cafe, if you have an idea for a topic you'd like to discuss in a breakout room, please add it below and put your name next to it. 
+
+If you like one of the topics that are already suggested, please add your name next to that one. For more information about breakout rooms see [the description on GitHub](https://github.com/alan-turing-institute/the-turing-way/blob/master/project_management/online-collaboration-cafe.md#breakout-rooms).
+
+* Main Room:
```
[the remark this answers is not in the document]
Comment (aleesteele) [reply governance/community-calls/20240221-collaboration-cafe.md:32]: Thanks @KirstieJane! I think I've been able to fix them – let me know if they're not rendering for you 😄

**Comment by aleesteele** [comment]
After one final review (where I found a few last rendering errors 😮‍💨 ), these are finally ready for another review! 

Updates on the names and archiving: After talking with our Data Protection officer, I re-added the names & check-in questions, and will be making to flag that more actively in future calls. Thanks for flagging Kirstie!

**Review by KirstieJane — APPROVED** [review]
Reviewed! Didn't look for typos but did look for text that shouldn't be shared on GitHub and didn't see anything 😸 

Thanks @aleesteele

**Pull request merged**